pkg/completion/completion.go

```diff
@@ -91,3 +91,15 @@ func AutocompleteSubgidName(cmd *cobra.Command, args []string, toComplete string
 func AutocompleteSubuidName(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
 	return autocompleteSubIDName("/etc/subuid")
 }
+
+// AutocompleteArch - Autocomplete architectures supported by container engines
+func AutocompleteArch(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+	completions := []string{"aarch64", "i686", "ppc64le", "s390x", "x86_64"}
```

Comment:
I think `x86_64` should be `amd64`

Reply:
Ok, I based my names on the distros supported by Fedora.  I am not sure if these names are solely based on golang names or arch names.  IE is there a list of arches that people actually use in container images available?

Reply:
Ok based on your link above I added the valid windows and linux arches as I beleive these are the ownly os/arches supported in the container world at this time.  Also added an autocompletePlatform with the same information.